packages/neuron-ui/src/locales/en.json

```diff
@@ -97,7 +97,7 @@
       },
       "actions": {
         "close": "Close",
-        "rescan": "Reconnect",
+        "reconnect": "Reconnect",
```

Comment:
The `key` `rescan` is not changed in codebase

Ref:
 `hardware-sign.actions.rescan`: https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-ui/src/components/HardwareSign/index.tsx#L438

Reply:
` https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-ui/src/components/HardwareSign/index.tsx#L438` is `hardware-sign.actions.rescan`, but not `hardware-verify-address.actions.rescan`

Reply:
I see, `hardware-verify-address.actions.reconnect` is used at https://github.com/nervosnetwork/neuron/blob/c60d9367cf2c7640df06799d2dfb2eb367a98ee4/packages/neuron-ui/src/components/VerifyHardwareAddress/index.tsx#L210